card/db/migrate_core_cards/20190823220018_cleanup_for_decko_1_0.rb

```diff
@@ -9,7 +9,7 @@ def up
       delete_card [perm, :right, :options]
     end
     delete_card "*missing link"
-    delete_card "tags"
+    # delete_card "tags"
```

Comment:
WikiRate has 8000+ `+tag` cards.  We don't want tag in the seed data, but I'm not sure we want to migrate it out of every site.

Reply:
right, we don't want that